packages/components/input-number/src/input-number.vue

```diff
@@ -317,7 +320,7 @@ onMounted(() => {
 })
 onUpdated(() => {
   const innerInput = input.value?.input
-  innerInput?.setAttribute('aria-valuenow', `${data.currentValue}`)
+  innerInput?.setAttribute('aria-valuenow', `${data.currentValue || ''}`)
```

Comment:
```suggestion
  innerInput?.setAttribute('aria-valuenow', `${data.currentValue ?? ''}`)
```
The value of `data.currentValue` may be `0`.